src/labels/CustomLabels.labels

```diff
@@ -3890,6 +3890,30 @@ To check your GAU Allocation settings, go to Donations | GAU Allocations.</value
         <shortDescription>error message if the donation matched opp(s), but behavior required no match</shortDescription>
         <value>The Donation information matched against one or more Opportunities, but the setting required no matches, so the NPSP Data Importer did not import it. The Donation Possible Matches field contains the Salesforce Ids of the matched Opportunities.</value>
     </labels>
+    <labels>
+        <fullName>bdiErrorExistingDonationAllocations</fullName>
+        <categories>Data Importer</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>bdiErrorExistingDonationAllocations</shortDescription>
+        <value>There are existing allocations on the donation which may conflict with the GAU allocations entered.</value>
```

Comment:
@sam-knox - Could you review these 3 labels?

Reply:
I mentioned sam on the ticket, and explained the use cases, these are placeholder until he has reviewed.

Reply:
@callen-sfdo - I posted some questions on the ticket but can post here too. Does this text accurately capture the errors being reported here?

1: "The provided GAU Allocations exceed 100 percent." 

2: "The existing GAU Allocations on the donation Opportunity record and the provided GAU Allocations exceed 100 percent." 

3: "The existing GAU Allocations on the donation Payment record and the provided GAU Allocations exceed 100 percent."

In cases 2 and 3, the errors occur only when the provided allocations exceed 100%, right? So if there was an existing Allocation on the donation at 50% and the DI had another at 50%, we shouldn't see the error?

Reply:
@sam-knox for 2 and 3 there is an issue whenever there are other existing allocations on the donation regardless of percent.

Reply:
@callen-sfdo - Ok thanks, then I'd like to revise messages 2 and 3 in a future work item. 

For the customer, I suppose they'd have to remove the existing allocations on either the opp/payment or from the DI and re-process.